### PR TITLE
Add short version of certificate and certificate request info

### PR DESCRIPTION
**Pull request by maraino**
### Description
This PR adds short information for certificates and certificate requests.

### CODE REVIEW

File: certformat.go [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+type certificateRequestShort struct {
+	PublicKeyAlgorithm string
+	Subject            string
+	SANs               []string
+}
+
+func newCertificateRequestShort(cr *x509.CertificateRequest) *certificateRequestShort {
+	var sans []string
+	for _, s := range cr.DNSNames {
```
Comment (dopey) [inline certformat.go]: How come you're not using `getSans()` here?

File: certformat.go [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+	}
+	return fmt.Sprintf("%s %s", algorithm, params)
+}
+
+func abbreviated(s string) string {
+	l := len(s)
+	if l <= 8 {
+		return s
+	}
+	return s[0:4] + "..." + s[l-4:]
```
Comment (dopey) [inline certformat.go]: [:4], but I'm nitpicking

File: certformat.go (hunk 1 above)
Comment (maraino) [reply certformat.go]: because they are different types

File: certinfo_test.go [hunk 3]
```diff
@@ -61,26 +61,79 @@ func testPair(t *testing.T, certFile, refFile string, inputType InputType) {
 	}
 }
 
+// Compares a PEM-encoded certificate to a refernce file.
```
Comment (dopey) [inline certinfo_test.go]: ... to reference file (spelling)

**Pull request merged**